### PR TITLE
Minor fixes to component docs

### PR DESCRIPTION
**Pull request by razor-x**
- Fix typo
- Fix broken links
- Always link to react.seam.co
- Minor copy updates to seam-components
- Normalize component docs

### CODE REVIEW

File: docs/seam-components/react-components.md [hunk 1]
```diff
@@ -111,16 +113,18 @@ The access code details component shows the properties, settings, and issues for
...
-The Supported Devices table allows you to filter, sort and view all the
+The supported device table allows you to filter, sort, and view all the
 devices compatible with Seam. You can use this table to indicate what
 devices are supported by your platform.
 
 ```ts
-<SupportedDeviceTable />
+<SeamProvider publishableKey="YOUR_PUBLISHABLE_KEY">
+  <SupportedDeviceTable />
+</SeamProvider>
```
Comment (seveibar) [inline docs/seam-components/react-components.md:127]: hmm, I realize you're bringing this in line with the rest of this page, but this page really shouldn't have SeamProvider usage at all imo- it doesn't have imports and SeamProvider can be used in different ways i.e. with client session or publishable key

**Review by seveibar — APPROVED** [review]

File: docs/seam-components/react-components.md (hunk 1 above)
Comment (razor-x) [reply docs/seam-components/react-components.md:127]: Maybe remove it from this page entirely and just have it on the guide for now?

Comment (razor-x) [reply docs/seam-components/react-components.md:127]: Oh, I see. Yes we should remove `SeamProvider` from all the component examples, agree!

Comment (razor-x) [reply docs/seam-components/react-components.md]: [`7741dba` (#87)](https://github.com/seamapi/api-docs/pull/87/commits/7741dba6fd13ebd5c8f9956840c333847d822ce2)

**Pull request merged**